.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -0,0 +1,74 @@
+name: publish-technical-documentation-release
+
+on:
+  push:
+    branches:
+      - <BRANCH GLOB>
+    tags:
+      - <TAG GLOB>
```

Comment:
For simplicity, I'd follow the same conventions as the Grafana Agent:https://github.com/grafana/agent/blob/main/.github/workflows/publish-documentation-versioned.yml

```yaml
  push:
    branches:
      - 'release-*'
    tags:
      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Reply:
Thanks, added in c5d68ec863d90cca13e8612ff8f76fbc3fdcee46